### PR TITLE
added links prisma schema

### LINKED ISSUE #2: Model: Link
**The model should contain the following:**

- title - string
- link - string
- slug - string (unique)
- description - string
- iconUrl - string
- isCustomIcon - boolean
- keywords - string[]

**Acceptance Criteria:**
- Prisma scheme is created and is usable

### PR DESCRIPTION
**Pull request by FearsomeRover**
Added the requested fields to the prisma model

### CODE REVIEW

**Comment by Tschonti** [comment]
Did we do the migration and the link resource (service, controller) generation on your machine? If so, please push those as well

**Comment by FearsomeRover** [comment]
Sorry, i forgot to add those in the first place. It should be good now.

**Review by Tschonti — CHANGES_REQUESTED** [review]
Nice! One small thing: please delete the `package-lock.json` file. It was probably generated because you used an npm command instead of yarn. In our case, `yarn.lock` holds the same information, so we don't need this file.
Also, please resolve the conflicts. Let me need if you need help with this, it might get ugly 😄

**Comment by FearsomeRover** [comment]
It hopefully should be good now!

File: packages/backend/src/app.module.ts [hunk 1]
```diff
@@ -1,9 +1,13 @@
+@Module({
+  imports: [LinksModule],
+=======
```
Comment (Tschonti) [inline packages/backend/src/app.module.ts]: Some remnants of the merge conflict - pls remove

**Review by Tschonti — CHANGES_REQUESTED** [review]
Just one more thing and it's good to go 🚀 
it's good practice to start the app after resolving a conflict to see if it compiles - soon this will be done automatically in Github actions when you submit a PR

File: packages/backend/src/app.module.ts (hunk 1 above)
Comment (FearsomeRover) [reply packages/backend/src/app.module.ts]: Sorry for the inconvenience.

**Review by Tschonti — APPROVED** [review]
Thanks for the work, I found a few more minor things that I overlooked before, didn't want to bother you anymore 😄 
While package-lock.json was not needed, package.json is. And the Link module was imported, but not used in app.module. When the app reaches a state where it can be manually tested, these issues will be more noticeable so no problem if stuff like this stays in the code for now. But this is why these huge conflicts are problematic, it's near impossible to resolve them perfectly and no automatic way of checking them.

**Pull request merged**